backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleCommandConfig.java

```diff
@@ -263,131 +257,91 @@ public AnsibleCommandConfig logFileSuffix(String logFileSuffix) {
-        try {
-            Files.copy(source, dest, StandardCopyOption.REPLACE_EXISTING);
-        } catch (Exception ex) {
-            throw new AnsibleRunnerCallException("Failed to copy project files into designated host execution directory", ex);
-        }
+        return tempProject;
     }
 
-
-    private void createInventoryFile() {
-        Path inventoryFile = null;
-        if (getInventoryFile() == null) {
-            // If hostnames are empty we just don't pass any inventory file:
-            if (CollectionUtils.isNotEmpty(hostnames())) {
-                try {
-                    inventoryFile = Files.createTempFile("ansible-inventory", "");
-                    Files.write(inventoryFile, StringUtils.join(hostnames(), System.lineSeparator()).getBytes());
-                    inventoryFile(inventoryFile);
-                } catch (Exception ex) {
-                    throw new AnsibleRunnerCallException(
-                        String.format(
-                            "Failed to create inventory file '%s':",
-                            inventoryFile == null ? null : inventoryFile.toString()), ex);
-                }
+    private void createHostFile(File inventory) {
+        File hostsFile = new File(String.format("%1$s/hosts", inventory));
+        if (CollectionUtils.isNotEmpty(hostnames())) {
+            try {
+                hostsFile.createNewFile();
+                Files.write(hostsFile.toPath(), StringUtils.join(hostnames(), System.lineSeparator()).getBytes());
+            } catch (IOException ex) {
+                throw new AnsibleRunnerCallException(
+                    String.format("Failed to create inventory file '%s':", hostsFile.toString()), ex);
             }
         }
     }
 
-    private void setVars() {
-        String vars = formatCommandVariables(variables);
-        extraVars = new File(String.format("%1$s/%2$s/env/extravars", AnsibleConstants.ANSIBLE_RUNNER_PATH, uuid));
+    private void createExtraVarsFile(File env) {
+        String vars = getFormattedVariables();
+        File extraVars = new File(String.format("%1$s/extravars", env));
         try {
             extraVars.createNewFile();
             Files.writeString(extraVars.toPath(), vars);
-        } catch(Exception ex) {
+        } catch(IOException ex) {
             throw new AnsibleRunnerCallException(
                 String.format("Failed to create host variables file '%s':", extraVars.toString()), ex);
         }
     }
 
-    protected String formatCommandVariables(Map<String, Object> variables) {
+    private String getFormattedVariables() {
         String result;
         try {
-            result = mapper.writeValueAsString(variables);
+            result = this.mapper.writeValueAsString(this.variables);
         } catch (IOException ex) {
             throw new AnsibleRunnerCallException("Failed to create host deploy variables mapper", ex);
         }
         return result;
     }
 
     public void cleanup() {
```

Comment:
Do we need this function at all?

Reply:
AFAIK no, I did not remove just in case we would need it in some edge case but can not think of any scenario